frontend/src/components/modals/InviteModal.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { Modal, MessageBoxFunctionalLayout } from 'wix-style-react';
+import { useSelector, useDispatch } from 'react-redux';
+import doAction from '../../state/actions/action';
+import InviteForm from '../auth/InviteForm';
+import Notification from '../Notification';
+import {
+  ACTION_FAILED, ACTION_SUCCEEDED,
+} from '../../constants/ActionStatus';
+
+const InviteModal = ({ isModalOpened, onCloseModal }) => {
+  const [showNotification, setShowNotification] = useState(false);
+
+  const dispatch = useDispatch();
+  const action = useSelector(state => state.action);
+
+  const inviteUser = user => {
+    dispatch(doAction('invites/invite', user));
```

Comment:
This seems like an interesting idea, couple of concerns with this approach:

- contract between components and redux seems cleaner when you don't pass path of an endpoint - you just declaratively state what you want, e.g. `dispatch(fetchInvites(user));` then it becomes a part of _state manager_ to be concerned about how to do it
- `doAction(...)` seems incredibly vague, this name says nothing for the 1st time you read it
I still think it's a good idea and we can perhaps come up with something based on this e.g. still have a declarative action, which would use this approach 'behind the scenes'